test/reline/yamatanooroti/test_rendering.rb

```diff
@@ -1054,8 +1060,8 @@ def test_simple_dialog_with_scroll_screen
           prompt>   2
           prompt>   3#
           prompt>   4
-          prompt>   5
-          prompt>   6 Ruby is...
+          prompt>   5 Ruby is...
+          prompt>   6 A dynamic, open source programming
```

Comment:
Why is this change needed?

Reply:
It's because it was testing wrong behavior.

![dialog_pos_bug](https://github.com/ruby/reline/assets/1780201/50dbf57a-0aa1-4b2c-8550-247205a2a9eb)
This bug is fixed in this pull request, so the expected state needs to be updated.